src/components/Output.tsx

```diff
@@ -469,7 +469,7 @@ const $Text = styled.output<{ withParentheses?: boolean }>`
     opacity: 0.5;
 
     &:after {
-      content: '-' var(--output-afterString);
+      content: '—' var(--output-afterString);
```

Comment:
.___, hahahha what changed lmao

Reply:
loool I think it renders as slightly larger on the web 😭 my old designer used to be super opinionated about this